continuous-delivery/pack.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+set -ex
+
+# force a failure if there's no tag
+git describe --tags
+# now get the tag
+CURRENT_TAG=$(git describe --tags | cut -f2 -dv)
+# convert v0.2.12-2-g50254a9 to 0.2.12
+CURRENT_TAG_VERSION=$(git describe --tags | cut -f1 -d'-' | cut -f2 -dv)
```

Comment:
May be check if the tagged commit at current head? This can prevent updating binaries without bumping versions.

Reply:
I removed a check that was there for testing. Added it back to check if the tag is release. So, in that case, we can prevent updating binaries without proper release, which will also prevent the version is not updated, I think.